app/models/ab_experiment.rb

```diff
@@ -0,0 +1,68 @@
+# This object wraps the FieldTest logic to provide a bit of insulation
+# between the FieldTest implementation and the application logic.
+#
+# This is a SimpleDelegator because the field_test method requires
+# controller context when the participating user is nil.
+#
+# Use the AbExperiment.get method to interact with this class.
+class AbExperiment < SimpleDelegator
+  # Don't allow someone to circumvent the .get class method.  I want
+  # to follow the Law of Demeter and require that we us the @api
+  # public
+  private_class_method :new
+
+  # @api public
+  #
+  # A convenience method to reduce the
```

Comment:
Question: Is there an explanation missing here?

Reply:
Good catch.